README.md

```diff
@@ -71,4 +71,10 @@ To run the server side unit tests in intelliJ, right click on the `AgentDiscover
 From the command line, use `mvn test`.
 
 ## Frontend Tests
-To run the front-end unit tests, from command line navigate to `AgentDiscoveries-Frontend` and then run npm test.
\ No newline at end of file
+To run the front-end unit tests, from command line navigate to `AgentDiscoveries-Frontend` and then run npm test.
+
+## End-to-End Tests
+In order to run the End-To-End tests locally the webserver needs to be active, so ensure you first run the application as described in the "Run" section.
+
+Once the server is online, you can run the tests from the command line using `mvn verify -P integration-test`. This can be run from either the root directory or AgentDiscoveries-Backend. It will target the address `http://localhost:8080` by default.
+If you need to run the tests on another address for any reason, use the command `mvn verify -P integration-test -Dtarget.Address="http://foo.bar"` with the relevant address.
```

Comment:
Here you have a capital A for address, whereas it's lowercase in the pom. Any idea if that matters?

Reply:
Good catch, it needs to be consistent yes!